ui/src/alerting/components/CheckHistoryControls.tsx

```diff
@@ -0,0 +1,33 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+// import BackToTopButton from 'src/eventViewer/components/BackToTopButton'
```

Comment:
stray comment- what do you think about making it a habit to look over your PR as if you were reviewing someone else's right before submitting? :)